crates/bevy_math/src/bounding/bounded3d/primitive_impls.rs

```diff
@@ -166,7 +166,8 @@ impl Bounded3d for Capsule {
     fn aabb_3d(&self, translation: Vec3, rotation: Quat) -> Aabb3d {
         // Get the line segment between the hemispheres of the rotated capsule
         let segment = Segment3d {
-            direction: Direction3d::from_normalized(rotation * Vec3::Y),
+            // SAFETY: Multipling a normalized vector (Vec3::Y) with a rotation returns a normalized vector.
+            direction: unsafe { Direction3d::new_unchecked(rotation * Vec3::Y) },
```

Comment:
Can `Direction(3d,2d)` implements rotation ?  (probably in another PR)
to directly use `rotation * Direction3d::Y` without unsafe.